tools/scripts/scrub_issues/scrub_issues.py

```diff
@@ -0,0 +1,319 @@
+# © 2021 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+
+# Examines code in ICU4C and ICU4J branches, finding lines with 'knownIssue' and
+# 'TODO'. Then uses JIRA API to check the status of each of the found issues.
+
+# Note that the "jira" module must be available to run this.
+
+# Types of results:
+#  non-JIRA issue ids that are not short forms of a JIRA id
+#  issues with a JIRA number but that don't start with 'ICU-' or 'CLDR-'
+#  JIRA issues that are resolved
+#  Unresolved JIRA issues found in code
+
+# Running this:
+#  python scrub_issues.py --icu_base <full path to "/icu/" on local computer
+#
+# option parameter 'detail_level' may be set to 'summary' to prevent output of
+#   lines in code for each issue.
+
+from jira import JIRA
+
+import argparse
+import logging
+import os.path
+import re
+import subprocess
+import sys
+
+# Globals
+jira = JIRA(server='https://unicode-org.atlassian.net')
+
+projects = jira.projects()
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('scrub issues')
+
+# Base for the ICU directory
+ICU_BASE = '/usr/local/google/home/ccornelius/ICU78/icu'
```

Comment:
Is there a way to generalize this?  If it's easier to just change the code, you should probably mention this in the README.